service/src/main/java/bio/terra/tanagra/api/accesscontrol/ResourceType.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.tanagra.api.accesscontrol;
+
+import static bio.terra.tanagra.api.accesscontrol.Action.*;
+
+import java.util.List;
+
+/** Names of Tanagra resources and the actions that are possible for each. */
+public enum ResourceType {
+  UNDERLAY(List.of(READ, QUERY_INSTANCES, QUERY_COUNTS)),
```

Comment:
It could be worth considering GCP style permissions (e.g. `tanagra.underlay.read`). Familiar and perhaps easier to work without.

Reply:
I think this is functionally equivalent, though maybe the naming could be made more similar. Both define a permission for an "action" on an "object" (e.g. READ an UNDERLAY), it's just specified with two enums rather than as a single string.